redisenterprise/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "RedisEnterprise",
+  "maintainer": "github@mague.com",
+  "manifest_version": "1.0.0",
+  "name": "redisenterprise",
+  "metric_prefix": "redisenterprise.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "",
+  "guid": "727dcbe6-9ed6-409f-ad72-265939b90da8",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "RedisEnterprise",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
```

Comment:
Also another note: leaving this to false means that the documentation page won't show this integration. You can change it later in another PR if this is what you want

Reply:
@hithwen Good point - I've changed it to true as we want this to be available